platform-apis/src/main/java/org/stellar/platform/apis/callbacks/responses/GetRateResponse.java

```diff
@@ -1,18 +1,41 @@
 package org.stellar.platform.apis.callbacks.responses;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.gson.annotations.SerializedName;
 import java.time.LocalDateTime;
 import lombok.Data;
 
 @Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
We are using gson as our preferred json mapper. I think this field may be ignored.
https://github.com/stellar/java-stellar-anchor-sdk/blob/main/anchor-reference-server/src/main/java/org/stellar/anchor/reference/AnchorReferenceServer.java#L20

Reply:
That's strange.... 

We probably should review whether to stay with gson or switch to jackson later. I know Jackson is pretty old and gson is more modern I did not spend too much time researching.